CMakeLists.txt

```diff
@@ -343,10 +343,56 @@ if (NOT ENABLE_CXX11)
 	if (ENABLE_STDCXX_SYNC)
 		message(ERROR "ENABLE_STDCXX_SYNC is set, but C++11 is disabled by ENABLE_CXX11")
 	endif()
+	if (DEFINED USE_CXX_STD)
+		message(ERROR "USE_CXX_STD can be set only when ENABLE_CXX11 is on")
+	endif()
```

Comment:
`STDCXX` and `CXX_STD`. Can these names be more similar?
Maybe `USE_STSCXX`?

Reply:
STDCXX is an internal variable used intermediately. The only problem with it could be if it's conflict-prone.